src/huggingface_hub/keras_mixin.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import os
+from pathlib import Path
+
+from huggingface_hub import ModelHubMixin, hf_hub_download
+
+
+logger = logging.getLogger(__name__)
+
+
+class KerasModelHubMixin(ModelHubMixin):
+
+    _CONFIG_NAME = "config.json"
+    _WEIGHTS_NAME = "tf_model.h5"
+
+    def __init__(self, *args, **kwargs):
```

Comment:
I assume the actual content of the `__init__` method is coming later, even if it's just a `super()` call!

Reply:
Resolved offline - no super call needed here since its a mixin 😄